lib/forklift/box_distributor.rb

```diff
@@ -111,6 +111,8 @@ def configure_networks(networks)
 
     def configure_ansible(machine, ansible, box_name)
       return unless ansible
+      return if !ansible.key?('group') || ansible['group'].nil?
```

Comment:
Could we just skip the group setup portion down below so the box is still setup and configured? Basically, just not require a group to be specified in all cases as I think thats a valid use case.